admin_manual/configuration_files/federated_cloud_sharing_configuration.rst

```diff
@@ -8,9 +8,15 @@ file shares between ownCloud servers, in effect creating a cloud of ownClouds.
 You can automatically send an email notification when you create the share, add 
 password protection, allow users to upload files, and set an expiration date.
 
+Federated cloud sharing now works a little differently than in ownCloud 8.0. 
```

Comment:
@carlaschroder well the UI is missing it from telling you, but it works.
we fixed the placeholder message in 8.1 only looks like.